### ISSUE TITLE
Update installation hints on CANVAS

### ISSUE THREAD

**Issue by Grothesk242** [iobroker-community-adapters/ioBroker.mihome-vacuum#651]
Please update installation instructions for canvas: 

Replace 
```
sudo npm install canvas --unsafe-perm=true
```
with a simple
```
npm install canvas
```
(of course to be executed in the correct directory) 

--unsafe-perm is deprecated in npm>6,

**Comment by dirkhe** [comment]
will be in one of the next release

**Comment by dirkhe** [comment]
Implemented in 3.7.0

**Comment by Grothesk242** [comment]
Please also remove the sudo. 
It will break the set up when npm is called with root power.

**Comment by dirkhe** [comment]
Ok, I will do it

**Comment by dirkhe** [comment]
fixed in 3.8.0
